docs/src/components/home/HomeEditor.tsx

```diff
@@ -8,36 +8,37 @@ import {
 } from 'react-live';
 import * as AUI from '@aws-amplify/ui-react';
 
-const { Flex, View, Alert } = AUI;
+const { View, Alert } = AUI;
 
 const code = `() => {
   const { tokens } = useTheme();
   return (
     <Card variation="elevated">
       <Flex direction="row" alignItems="flex-start">
```

Comment:
nit: `direction="row" alignItems="flex-start"` are the defaults for Flex, so these could be removed. As well as for the other Flex's on this page that aren't set to `direction="column"`

Reply:
`direction="row"` is the default, but the default `alignItems` is `stretch`